tests/integration/targets/setup_rabbitmq/tasks/ubuntu.yml

```
@@ -1,53 +1,93 @@
 ---
-# https://www.rabbitmq.com/install-debian.html#apt-pinning
-- name: Pin erlang version that rabbitmq supports
-  copy:
-    dest: /etc/apt/preferences.d/erlang
-    content: |
-        Package: erlang*
-        Pin: version 1:20.3.8.18-1
-        Pin-Priority: 1000
 
-        Package: esl-erlang
-        Pin: version 1:20.3.6
-        Pin-Priority: 1000
+# https://stackoverflow.com/questions/25193161/chfn-pam-system-error-intermittently-in-docker-hub-builds/25267015
+- name: Disable chfn
+  file:
+    path: /usr/bin/chfn
+    src: /bin/true
+    state: link
+    force: yes
+
+- name: Install Essential Dependencies
+  apt:
+    name:
+      - curl
+      - gnupg
+      - debian-keyring
+      - debian-archive-keyring
+      - python3-apt
+    state: present
 
 - name: Install https transport for apt
   apt:
     name: apt-transport-https
-    state: latest
+    state: present
     force: yes
 
-- name: Add RabbitMQ release signing key
+- name: Team RabbitMQ's main signing key
+  apt_key:
+    keyserver: hkps://keys.openpgp.org
+    id: 0A9AF2115F4687BD29803A206B73A36E6026DFCA
+
+# https://www.rabbitmq.com/install-debian.html#apt-cloudsmith
+#
+- name: Cloudsmith - modern Erlang repository
+  apt_key:
+    url: https://dl.cloudsmith.io/public/rabbitmq/rabbitmq-erlang/gpg.E495BB49CC4BBE5B.key
+    state: present
+
+- name: Cloudsmith - RabbitMQ repository
   apt_key:
-    url: https://ansible-ci-files.s3.amazonaws.com/test/integration/targets/setup_rabbitmq/rabbitmq-release-signing-key.asc
+    url: https://dl.cloudsmith.io/public/rabbitmq/rabbitmq-server/gpg.9F4587F226208342.key
     state: present
 
 - name: Add RabbitMQ Erlang repository
   apt_repository:
-    repo: ppa:rabbitmq/rabbitmq-erlang
+    repo: deb https://dl.cloudsmith.io/public/rabbitmq/rabbitmq-erlang/deb/ubuntu bionic main
     filename: 'rabbitmq-erlang'
     state: present
-    update_cache: yes
+      #update_cache: yes
+    update_cache: no
```

why did you disable update_cache? This might be (part of) why it's getting so many package conflicts.
```suggestion
    update_cache: yes
```